app/client/components/HelpCentrePage.tsx

```diff
@@ -0,0 +1,49 @@
+import { css, Global } from "@emotion/core";
+import { Redirect, Router } from "@reach/router";
+import React, { lazy, Suspense } from "react";
+import { fonts } from "../styles/fonts";
+import global from "../styles/global";
+import { AnalyticsTracker } from "./analytics";
+import { CMPBanner } from "./consent/CMPBanner";
+import HelpCentrePageSkeleton from "./HelpCentrePageSkeleton";
+import { Main } from "./main";
+import { ScrollToTop } from "./scrollToTop";
+
+const HelpCentre = lazy(() =>
+  import(/* webpackChunkName: "HelpCentre" */ "./helpCentre/helpCentre")
```

Comment:
can we write a sneaky comment somewhere explaining that the webpackChunkName comments here are not just comments but functional

👍

Reply:
Yup, I will add those. Also added information about magic comments to the PR description.

Reply:
Done!